libraries/AP_HAL_Linux/Util.h

```diff
@@ -9,6 +9,12 @@
 #include "ToneAlarmDriver.h"
 #include "Semaphores.h"
 
+enum hw_type {
+    HARDWARE_TYPE_BEBOP = 0,
+    HARDWARE_TYPE_BEBOP2,
+    NUM_HARDWARES,
+};
```

Comment:
Ok, I had missed this one. This implementation is generic and I will bring get_rpi_version to the Util class instead. Thanks.